README.md

```diff
@@ -12,6 +12,50 @@ A subset of available Janus IDP plugins is available at our [community site](htt
 
 You can also see the [Plugin Marketplace](https://backstage.io/plugins) for other open-source plugins you can add to your Backstage instance.
 
+## Dynamic Plugins Installation
+
+It is possible to install plugins without code changes in a backstage that supports [Dynamic Plugins](https://github.com/janus-idp/backstage-showcase/blob/main/showcase-docs/dynamic-plugins.md) (e.g. [Janus IDP](https://janus-idp.io/) and [Red Hat Developer Hub](https://developers.redhat.com/rhdh)). Follow the steps below to install a dynamic plugin:
+
+- Map the dynamic plugins root directory in `app-config.yaml`:
```

Comment:
`app-config.local.yaml`, sorry missed this one before commenting on the other one :-)